nvm.sh

```diff
@@ -2277,12 +2307,16 @@ nvm_download_artifact() {
     nvm_err "Removing the broken local cache..."
     command rm -rf "${TARBALL}"
   fi
+
   nvm_err "Downloading ${TARBALL_URL}..."
-  nvm_download -L -C - "${PROGRESS_BAR}" "${TARBALL_URL}" -o "${TARBALL}" || (
+  download_result=$(nvm_download_v2 -L -C - "${PROGRESS_BAR}" "${TARBALL_URL}" -o "${TARBALL}")
+  nvm_err "Download Http Status ${download_result}"
+  # shellcheck disable=SC3010
+  if [[ ${download_result} != "200" ]]; then
```

Comment:
this is bash-specific syntax which will break nvm.
```suggestion
  if [ "${download_result}" != '200' ]; then
```